canonical/ubuntu.com
# Rename file to fix pricing/desktops navigation

## Done

Fix pricing/desktops navigation

## QA

Go to https://ubuntu-com-12503.demos.haus/pricing and click on the desktops navigation, it should work now with this fix

## Issue / Card

Fixes https://github.com/canonical/ubuntu.com/issues/12501


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
